core/controllers/collection_viewer.py

```diff
@@ -80,10 +80,12 @@ def get(self, collection_id):
 class CollectionDataHandler(base.BaseHandler):
     """Provides the data for a single collection."""
 
+    @require_collection_playable
```

Comment:
Added require_collection_playable to this handler, is this a requirement for readonly access to collection data?